packages/renderer/src/lib/extensions/ExtensionDetailsReadme.spec.ts

```diff
@@ -19,15 +19,19 @@
 import '@testing-library/jest-dom/vitest';
 
 import { render, screen } from '@testing-library/svelte';
+import { tick } from 'svelte';
 import { beforeEach, expect, test, vi } from 'vitest';
 
 import ExtensionDetailsReadme from './ExtensionDetailsReadme.svelte';
 
 async function waitRender(customProperties: object): Promise<void> {
-  const result = render(ExtensionDetailsReadme, { ...customProperties });
-  while (result.component.$$.ctx[0] === undefined) {
-    await new Promise(resolve => setTimeout(resolve, 100));
-  }
+  render(ExtensionDetailsReadme, { ...customProperties });
+  await tick();
+  await tick();
+  await tick();
+  await tick();
+  await tick();
+  await tick();
```

Comment:
why

Reply:
onMount is asynchronous so by default after a call to the render method the onMount method may not be completed

before we were monitoring the $$ctx object to see if some variables were defined (and $$ctx is no longer there with svelte5) and the recommended way is to tick

for this specific component, as there is a lot of different context switching in the onMount method I had to tick the number of times to have the component being ready